charm/ksql/lib/charms/layer/ksql.py

```diff
@@ -25,21 +25,30 @@
 from charmhelpers.core.templating import render
 from charmhelpers.core.hookenv import config
 
-KSQL_PORT=8088
```

Comment:
I think it's better to leave this hard-coded. Don't really have a use case for changing the port.